pages/vote/[voteId].js

```diff
@@ -0,0 +1,74 @@
+import { Footer } from "../../components/Footer";
+import { Header } from "../../components/Header";
+import React from "react";
+import { ResultsChart } from "../../components/ResultsChart";
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import { useRouter } from "next/dist/client/router";
+import { CreateLink } from "../../components/CreateLink";
+import { VoteForGift } from "../../components/VoteForGift";
+
+export default function Vote() {
+  const router = useRouter();
+  const voteId = router.query.voteId;
+
+  const [giftVote, setGiftVote] = useState([]);
+  const [url, setUrl] = useState();
+
+  const onVoteClick = (updatedGiftVote) => {
+    setGiftVote(updatedGiftVote);
+  };
+
+  const fetchGiftVote = async () => {
+    try {
+      const response2 = await fetch(`/api/giftVote/${voteId}`);
+      const newGiftVote = await response2.json();
+      setGiftVote(newGiftVote.vote);
+    } catch (error) {
+      console.error(error.message);
+    } finally {
+    }
```

Comment:
You can get rid of an empty finally branch
```suggestion
    }
```

Reply:
i removed this empty brackets